requirements/base.txt

```diff
@@ -1,2 +1,9 @@
-# Base requirements are empty. All of our minimum requirements are specified in setup.py's install_requires variable!
--e .
+django-filter==0.13.0
+django-js-reverse==0.7.2
+six==1.10.0
+djangorestframework==3.3.3
+django==1.9.7
+colorlog==2.6.0
+docopt==0.6.2
+django-mptt==0.8.5
+https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip
```

Comment:
Ah, I get it now... the wheel version is somehow not py2 compatible.. so we would need a way to force pip to use the tar.gz version which isn't possible to express in requirements.txt or install_requires... I guess pointing to the github tar.gz is the way to go for now then.

Anyways, better bump the version to 7.1.0 immediately so we don't fall behind!